src/ray/util/tests/BUILD

```diff
@@ -239,6 +239,8 @@ ray_cc_test(
     size = "small",
     tags = [
         "team:core",
+        # TSAN fails to understand synchroization logic, and the test suite always contains exactly one test case so we disable the whole test suite.
```

Comment:
Can you add more details: e.g. tsan thinks we will call flush on the same ostream concurrently (one from main, one from pipe dump thread) etc and this won't happen before balabaha.

Add as many details as you can since it will help people look at it in the future.

Reply:
sure